plugins/ircbridge/ircbridge.js

```diff
@@ -8,6 +8,19 @@ var api, config, log, persist, storage, emoji;
 var handlers = {}, clients = {}, groupScopeMap = {}, channelScopeMap = {};
 var inboundRoutes = [], outboundRoutes = [], copyRoutes = [], readyServers = [];
 
+const YES = ['1', 'true', 'on'];
+const NO = ['0', 'false', 'off'];
+
+const PREFERENCES_KEY_RELAYABLE = 'relayable';
+const PREFERENCES_KEY_COLOUR = 'colour';
+const PREFERENCES_KEY_ALIAS = 'alias';
```

Comment:
I generally dislike this, but since js has no symbols and I don't have a better opinion. Meh.

Why not refactor it into a preferences object? You'd have method names instead of ambiguous constants/strings :)

Reply:
Effort 🤷 This is more consistent with that's already there.

Reply:
You mean consistent like injecting random ES6 in a 100% ES5 codebase? :clown_face: